flowcraft/generator/templates/momps.nf

```diff
@@ -75,17 +76,18 @@ process momps_report_{{ pid }} {
     publishDir "results/typing/momps_{{ pid }}/", pattern: "*.tsv"
 
     input:
-    file(st_file) from momps_st_{{ pid }}.collect()
-    file(profile_file) from momps_profile_{{ pid }}.collect()
+    val sample_id from momps_sample_id_{{ pid }}
```

Comment:
The same login applies here. If output is not being compile, then it makes more sense to just create these files on the main process and move the publishDir there.